tests/integration/models/billing.spec.coffee

```diff
@@ -0,0 +1,257 @@
+_ = require('lodash')
+m = require('mochainon')
+Promise = require('bluebird')
+
+{ resin, givenLoggedInUser, IS_BROWSER } = require('../setup')
+
+eventuallyExpectProperty = (promise, prop) ->
+	m.chai.expect(promise).to.eventually.have.property(prop)
+
+describe 'Billing Model', ->
+
+	givenLoggedInUser()
+
+	hasActiveBillingAccount = false
+
+	withoutABillingAccountIt = (description, testFn) ->
+		it description, ->
+			if hasActiveBillingAccount
+				return this.skip()
+			testFn.apply(this, arguments)
```

Comment:
I think the `given` case is good, but this `without` behaviour isn't quite what there yet.

The issue is that here we end up here with tests that aren't run on the CI server. That means a user testing with a billing account locally won't notice if they fail, and neither will the CI server, and so somebody else later on will turn up and find everything broken (or worse: we'll have released the broken code and broken everything for real free users).

We should aim to be running _all_ the tests in CI, and only a subset locally for dev users that don't have a paid billing code set up.

I'm hoping there's a route through here where if you already have an account, we can use `updateInfo()` to remove it, run the free tests, and then put it back again afterwards. Does that sound plausible? This is a little tricky I know, ping me if you want to chat more about it.